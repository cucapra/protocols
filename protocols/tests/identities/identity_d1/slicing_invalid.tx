// ARGS: --verilog identities/identity_d1/identity_d1.v --protocol identities/identity_d1/identity_d1.prot
// RETURN: 101
slicing_invalid(1, 1);