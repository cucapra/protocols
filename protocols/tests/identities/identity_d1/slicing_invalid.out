error: Invalid slice operation: [15:31] on width 32
   ┌─ identities/identity_d1/identity_d1.prot:44:13
   │
44 │   assert_eq(s[15:31], DUT.s[15:31]); // illegal slices!
   │             ^ Invalid slice operation: [15:31] on width 32

